lib/patchelf/patcher.rb

```diff
@@ -20,12 +20,25 @@ class Patcher
     #   Filename of input ELF.
     # @param [Boolean] logging
     #   Whether to use a (stderr-initialized) logger for errors
```

Comment:
add statements that this option is deprecated

Reply:
placing @deprected  on same indent level as @param marks the constructor as deprecated.
Adjusted by placing a  it at the beginning of the description.
what's the proper way to do this ?

Reply:
Yap I don't find a keyword for deprecating a parameter either.

ITOW please say `Deprecated: use +on_error+ instead` here.